### ISSUE TITLE
App specific macros

### ISSUE THREAD

**Issue by cwagner22** [nikitavoloboev/km-macros#1]
Can you share your app specific macros too? Currently there is only global one. Ty

**Comment by nikitavoloboev** [comment]
Yeah, will do soon. There's quite a lot of macros that I have yet to add here. 🙂
